File: 2586. Count the Number of Vowel Strings in Range/solution.py
class Solution(object):
  def vowelStrings(self, words, left, right):
    """
    :type words: List[str]
    :type left: int
    :type right: int
    :rtype: int
    """
    vowels = {'a', 'e', 'i', 'o', 'u'}
    count = 0
    for word in words[left:right + 1]:
      if word[0] in vowels and word[-1] in vowels:
        count += 1
    return count

File: 2586. Count the Number of Vowel Strings in Range/solution.java
class Solution {
  public int vowelStrings(String[] words, int left, int right) {
    Set<Character> vowels = new HashSet<Character>(Arrays.asList('a', 'e', 'i', 'o', 'u'));
    int count = 0;
    for (int index = left; index < right + 1; index++){
      String word = words[index];
      if (vowels.contains(word.charAt(0)) && vowels.contains(word.charAt(word.length() - 1))){
        count += 1;
      }
    }
    return count;
  }
}